apps/ledger-live-desktop/src/renderer/init.jsx

```diff
@@ -127,6 +129,10 @@ async function init() {
 
   let accounts = await getKey("app", "accounts", []);
   if (accounts) {
+    // preload currency that's not in accounts list
+    if (!accounts.some(a => a.currency.id === "ethereum")) {
```

Comment:
don't we want to load the ERC20 even if user don't have accounts so they can see the list of erc20?

Reply:
Yeah totally, but maybe it's some clarity issue on my code